src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ShowPlan/Node.cs

```diff
@@ -704,6 +704,40 @@ private string GetRowStatisticsDisplayString(ulong? actualRows, double? estimate
             return SR.ActualOfEstimated(actualString, estimateString, percent);
         }
 
+        public string GetRowCountDisplayString() {
+            var actualRowsCounters = this[NodeBuilderConstants.ActualRows] as RunTimeCounters;
+            ulong? actualRows = actualRowsCounters != null ? actualRowsCounters.TotalCounters : (ulong?)null;
+            if(actualRows!= null){
+                return actualRows.Value.ToString();
+            }
+            var estimateRows = this[NodeBuilderConstants.EstimateRows] as double?;
+            var estimateExecutions = this[NodeBuilderConstants.EstimateExecutions] as double?;
+
+            if (estimateRows != null)
+            {
+                if (estimateExecutions != null)
+                {
+                    estimateRows = estimateRows * estimateExecutions;
+                }
+                // we display estimate rows as integer so need round function
+                estimateRows = Math.Round(estimateRows.Value);
+            } 
+            return estimateRows == null ? "" : estimateRows.Value.ToString();
+        }
+
+        public string GetNodeCostDisplayString() {
+            double cost = this.RelativeCost * 100;
+            string costText = "";
+            if (!this.HasPDWCost || cost > 0)
+            {
+                if (this.graph != null && this.graph.NodeStmtMap.Count < Node.LargePlanNodeCount)
+                {
+                    cost = Math.Round(cost);
+                }
+                costText = cost.ToString("0.##")+"%";
```

Comment:
Nit: Put a space between the "+" concatenation operator. It looks odd in it's current format.